### DISCUSSION TITLE
Duda [evento 5 parte 2] Give puffles

### DISCUSSION THREAD

**Discussion by mon-b** [IIC2133-PUC/2024-2#14] [Tarea 0] [answered]
Si el pingüino 2 puede recibir los puffles del rango del inventario pingüino 1, ¿los puffles recibidos quedan al final del inventario del pingüino 2? o debo acomodarlos de alguna manera relacionada con el rango original?

**Comment by 4n1t4** [comment, marked as answer]
Quedarían al final :)

**Reply by mon-b** [reply]
Gracias :))
